installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,76 @@
+[id="preparing-to-install-on-vsphere"]
+= Preparing to install on vSphere
+include::modules/common-attributes.adoc[]
+:context: preparing-to-install-on-vsphere
+
+toc::[]
+
+
+[id="preparing-to-install-on-vsphere-prerequisites"]
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+
+* You provisioned xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] for your cluster. To deploy a private image registry, your storage must provide
+`ReadWriteMany` access modes.
+
+* If you use a firewall and plan to use Telemetry, you
```

Comment:
For consistency in each of the "Preparing to install" pages, we could review if we want to duplicate these more detailed requirements for installation in this page, given that they are already covered in each of the installation assemblies.

Reply:
Agreed. This was resolved with the prior deletion.